tests/test_rocksdb_options.rs

```diff
@@ -132,3 +132,16 @@ fn test_set_wal_opt() {
     drop(db);
 }
 
+#[test]
+fn test_create_info_log() {
+    let path = TempDir::new("_rust_rocksdb_test_create_info_log_opt").expect("");
+    let mut opts = Options::new();
```

Comment:
yes, but here we don't need it in our test. The main purpose here is to create the LOG in another place with "LOG" name. If you want to test other log options, we should use other test.